src/Utils/Html.php

```diff
@@ -363,17 +363,39 @@ public function getText()
 
 
 	/**
+	 * @deprecated
+	 */
+	public function add($child)
+	{
+		trigger_error('Method Nette\Utils\Html::add() is deprecated, use addHtml() or addText() instead.', E_USER_DEPRECATED);
+		return $this->addHtml($child);
+	}
+
+
+	/**
 	 * Adds new element's child.
 	 * @param  Html|string Html node or raw HTML string
 	 * @return self
 	 */
-	public function add($child)
+	public function addHtml($child)
```

Comment:
You would need to wrap it as an Html instance. But I think that the current solution is better.